src/llmcompressor/modifiers/quantization/calibration.py

```diff
@@ -124,8 +124,22 @@ def call_observer(
             updated_scale, updated_zero_point = observer(
                 value, g_idx=g_idx, global_scale=global_scale
             )
-            update_parameter_data(module, updated_scale, f"{base_name}_scale")
-            update_parameter_data(module, updated_zero_point, f"{base_name}_zero_point")
+            # register or update scale & zero_point parameters (supports block shapes)
+            scale_name = f"{base_name}_scale"
+            zp_name = f"{base_name}_zero_point"
+            for name, value in [
```

Comment:
This shouldn't be required? If the original name or shape doesn't match, then there's a bug upstream